api/src/wfl/module/covid.clj

```diff
@@ -58,16 +31,27 @@
       (throw (ex-info (str "No such record") {:id id :table table})))
     record))
 
-;; interfaces
-;; queue operations
-(defmulti peek-queue!
-  "Peek the first object from the `queue`, if one exists."
-  (fn [queue] (:type queue)))
-
-(defmulti pop-queue!
-  "Pop the first object from the `queue`. Throws if none exists."
-  (fn [queue] (:type queue)))
+(defn peek-queue!
+  "Get first unconsumed item in DETAILS table."
+  [{:keys [details] :as _queue}]
+  (let [query "SELECT * FROM %s WHERE NOT consumed ORDER BY id ASC LIMIT 1"]
+    (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
+      (->> (format query details)
+           (jdbc/query tx)
+           (first)))))
```

Comment:
`WHERE consumed IS NULL`?  Or does SQL agree with Clojure on truthiness?

Reply:
Thanks for catching.  This is a holdover from when we thought we'd store this value as a boolean: 

https://www.postgresqltutorial.com/postgresql-boolean/

Will fix.